src/Config/ConfigInterface.php

```diff
@@ -37,6 +37,9 @@ public function getCookiePath();
     public function setCookieDomain($cookieDomain);
     public function getCookieDomain();
 
+    public function setCookieSameSite($cookieSameSite);
```

Comment:
adding methods to an interface is a BC break.
We might consider adding a new interfaces.
For example:

- `SameSiteCookieAwareInterface` which extends `SameSiteCookeiCapableInterface`
- `SameSiteCookieCapableInterface` which extends `ConfigInterface`

After this, we can work-around the BC break by checking for these interfaces and trigger exceptions in case someone configures the new configuration parameter but does not implement the new interface.

We can implement the ` in the `SameSiteCookieAwareInterface` in `StandardConfig` and should be good to go.

What do you think?